packages/apps/dev-wallet/src/pages/create-profile/create-profile.tsx

```diff
@@ -61,57 +70,64 @@ export function CreateProfile() {
       />
     );
   }
+
   return (
     <>
       <AuthCard backButtonLink="/select-profile">
```

Comment:
I think this is too general,
when the user is in personalise profile step `back button` should go to the enter password step not the select-profile page.
I think best place for handeling this is the stepper component, so I will approve this for now